pquery-run.sh

```diff
@@ -1352,13 +1358,21 @@ pquery_test(){
         SQL_FILE="--infile=${INFILE}"
       else
         # Multi-threaded run using a chunk from INFILE (${THREADS} clients)
-        echoit "Taking ${MULTI_THREADED_TESTC_LINES} lines randomly from ${INFILE} as testcase for this multi-threaded trial..."
+        if [ "$PQUERY_TOOL_NAME" == "pquery3-ps" ]; then
+          if [ ${TRIAL} -eq 1 ]; then
+            echoit "Creating metadata randomly using random seed..."
+          else
+            echoit "Loading metadata from ${WORKDIR}/step_$((${TRIAL}-1)).dll ..."
+          fi
+        else
+          echoit "Taking ${MULTI_THREADED_TESTC_LINES} lines randomly from ${INFILE} as testcase for this multi-threaded trial..."
+        fi
         shuf --random-source=/dev/urandom ${INFILE} | head -n${MULTI_THREADED_TESTC_LINES} > ${RUNDIR}/${TRIAL}/${TRIAL}.sql
         SQL_FILE="--infile=${RUNDIR}/${TRIAL}/${TRIAL}.sql"
       fi
-      # Debug echo "-------"; cat ${RUNDIR}/${TRIAL}/${TRIAL}.sql; echo "-------"
       if [[ ${PXC} -eq 0 && ${GRP_RPL} -eq 0 ]]; then
-        ${PQUERY_BIN} ${SQL_FILE} --database=test --threads=${THREADS} --queries-per-thread=${QUERIES_PER_THREAD} --logdir=${RUNDIR}/${TRIAL} --log-all-queries --log-failed-queries --user=root --socket=${RUNDIR}/${TRIAL}/socket.sock >${RUNDIR}/${TRIAL}/pquery.log 2>&1 &
+        echo "${PQUERY_BIN} ${SQL_FILE} --database=test --threads=${THREADS} --queries-per-thread=${QUERIES_PER_THREAD} --logdir=${RUNDIR}/${TRIAL} --log-all-queries --log-failed-queries --user=root --socket=${RUNDIR}/${TRIAL}/socket.sock --seed $RANDOMD --step ${TRIAL} --metadata-path ${WORKDIR}/ --seconds ${PQUERY_RUN_TIMEOUT} ${DYNAMIC_QUERY_PARAMETER}";
```

Comment:
do not enable log-all-queries by default. let it pass through DYNAMIC_QUERY_PARAMETER

Reply:
if seed option is not provided by the user then use a random seed.

Reply:
Done